Data/Contexts/ProductsDbContext.cs

```diff
@@ -0,0 +1,48 @@
+﻿using DomainModels;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data
+{
+    public class ProductsDbContext : DbContext
+    {
+        public ProductsDbContext(DbContextOptions<ProductsDbContext> options)
+            : base(options)
+        {
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (this.Database.GetPendingMigrations().Any())
+            {
+                this.Database.Migrate();
+            }
+        }
+
+        public virtual DbSet<Category> Categories { get => this.Set<Category>(); }
```

Comment:
Why not use just { get; } ?)

Reply:
Cause Visual Studio say, that here may be null.